apps/examples/src/examples/external-content-sources/ExternalContentSourcesExample.tsx

```diff
@@ -65,3 +71,26 @@ export default function ExternalContentSourcesExample() {
 		</div>
 	)
 }
+
+/* 
+Introduction:
+This example shows how to handle content pasted from external sources, this could be
+embeds, files, svgs, text, images, or urls. In this case we will handle text/html content.
+
+[1]
+We want to render our html on the canvas, the best way to do that is to create a new shape util.
+Here's where we define the type for our shape.
+
+[2]
+This is our shape util. It's a class that extends BaseBoxShapeUtil. For a more detailed
+example of how to create a custom shape, see the custom config example.
+
+[3]
+Where the action is. We use the onMount prop to get access to the editor instance via 
```

Comment:
```suggestion
We use the onMount prop to get access to the editor instance via 
```

Nit here: avoid idioms(?) like this, they tend to be missed by non-native english readers. Writing docs like this can be sometimes kiki without /idioms to boubize moving from topic to topic, but it's those can be sneakily kiki for folks asking "what is the action?"